packages/cli/src/services/service.deps.ts

```diff
@@ -0,0 +1,81 @@
+import { execa } from 'execa';
+import fs from 'fs';
+import path from 'path';
+
+import fsPromises from 'fs/promises';
+
+export class DepsService {
+  private packageManager: string;
+
+  constructor() {
+    this.packageManager = this.getPackageManager();
+  }
+
+  private findLockFile(dir: string): string | null {
+    const lockFiles = ['pnpm-lock.yaml', 'package-lock.json', 'yarn.lock'];
+    for (const file of lockFiles) {
+      if (fs.existsSync(path.join(dir, file))) {
```

Comment:
we use `sync` and `async` versions of the `fs` modules. I believe we could benefit from just sync or just async.